src/nuclio/projects/project/functions/version/version-configuration/tabs/version-configuration-build/version-configuration-build.component.js

```diff
@@ -174,6 +180,19 @@
         // Private methods
         //
 
+        /**
+         * Escapes single quotes in array of string
+         * Example:
+         * ['o'ne', 'tw''o', 'three''] => ['o\u0027ne', 'tw\u0027\u0027o', 'three\u0027']
+         * @param {Array} data - array of strings
+         * @returns {Array}
+         */
+        function escapeSingleQuote(data) {
+            return lodash.map(data, function (item) {
+                return item.replace(/'/g, '\\u0027');
```

Comment:
According to [YAML spec](https://yaml.org/spec/current.html#id2534365), I think the correct way to escape is to use `''`, not `\\u0027'`.